docker-compose.dev.yml

```diff
@@ -9,8 +9,6 @@ services:
       - "3000:3000"
     environment:
       - PUBLIC_PATH=http://${SITE_URL:-localhost}:3000/static/
-    volumes:
-      - dev_node_modules:/app/node_modules/
```

Comment:
We don't need this. It causes problems on macOs, and is not very useful in any case